tests/scattering2d/test_torch_scattering2d.py

```diff
@@ -336,11 +337,6 @@ def test_fft_exceptions(self, backend_device):
                         inverse=False)
         assert 'done with an inverse' in record.value.args[0]
 
-        x = torch.rand(4, 4, 1)
-        x = x.to(device)
-        with pytest.raises(TypeError) as record:
-            backend.fft(x)
-        assert 'complex' in record.value.args[0]
```

Comment:
Why no test?

Reply:
We assume that all tensors with a final dimension of 2 are complex; if not, they are real.

Reply:
do we have an answer for this?

Reply:
you have a  point